src/main/java/org/broadinstitute/consent/http/mail/message/NewResearcherLibraryRequestMessage.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import com.sendgrid.helpers.mail.Mail;
+
+import javax.mail.MessagingException;
+import java.io.Writer;
+
+public class NewResearcherLibraryRequestMessage extends MailMessage{
+
+    private final String NEW_RESEARCHER = "New Library Card Request in DUOS";
+
+    public Mail newResearcherLibraryRequestMessage(String toAddress, String fromAddress, Writer template) throws MessagingException {
+        return generateEmailMessage(toAddress, fromAddress, template, null, null);
+    }
+
+    @Override
+    String assignSubject(String referenceId, String type) {
+        return NEW_RESEARCHER;
+    }
```

Comment:
Not your code, so this is not a request for a change. I'm not thrilled with this pattern of passing in nulls and overriding `assignSubject`. We really should refactor this to be less brittle.